src/main/java/Opus.java

```diff
@@ -0,0 +1,76 @@
+import java.util.Scanner;
+
+public class Opus {
+
+    private Storage storage;
+    private TaskList taskList;
+    // private Ui ui;
+
+    public Opus(String filePath) {
+        // ui = new Ui();
+        storage = new Storage(filePath);
+        taskList = new TaskList(storage.load());
+    }
+
+    public void run() {
```

Comment:
Could this method be refactored into smaller methods to handle the different scenarios more effectively?